posts/2023/10/hpy_meetup.md

```diff
@@ -0,0 +1,135 @@
+<!--
+.. title: HPy meetup and CPython core dev sprint in Brno (Oct 7-10, 2023)
+.. slug: hpy-meetup-2023
+.. date: 2023-10-10 10:00:00 UTC
+.. author: fangerer
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+We just ended our rather informal HPy meetup in Brno (Czech Republic) which
+started on Saturday (October 7th) and lasted until Tuesday (October 10th).
+
+The main topics of the HPy meetup were:
+
+- The CPython Core Dev Sprint
+- Cython/HPy
+
+<!--TEASER_END-->
+
+# The CPython Core Dev Sprint
+
+Petr Viktorin, one of the most active CPython core developers, invited us to
```

Comment:
"Active" is arguable. I am the one most active in HPy, I guess :)
Could you remove it?
```suggestion
Petr Viktorin, one of the CPython core developers, invited us to
```